source/extensions/filters/http/lua/lua_filter.h

```diff
@@ -425,7 +520,8 @@ class Filter : public Http::StreamFilter, Logger::Loggable<Logger::Id::lua> {
   Http::FilterDataStatus doData(StreamHandleRef& handle, Buffer::Instance& data, bool end_stream);
   Http::FilterTrailersStatus doTrailers(StreamHandleRef& handle, Http::HeaderMap& trailers);
 
-  FilterConfigConstSharedPtr config_;
+  FilterConfigConstSharedPtr config_{};
+  PerLuaCodeSetup* per_lua_code_setup_{};
```

Comment:
Do we really need to have this as a member?

Reply:
We can discard this member, but we must move some code from doHeaders to decode/encodeHeaders.

Reply:
Oh, OK. Let's keep it then.

Reply:
emmmm, it would be nice if I saw your comment earlier. I have modified and updated the commit. orz

Reply:
But there are only two or three lines of redundant code, which I think should be acceptable.  I will wait until you review it again, and see if I need to change it back.